Need Help, how to fix mypy error while pre-commit

I am unable to commit code changes due to the following Mypy errors. Is there a solution to resolve this?

src\neo4j\_sync\work\session.py:549: error: ParamSpec components are not allowed here  [valid-type]
src\neo4j\_sync\work\session.py:550: error: ParamSpec components are not allowed here  [valid-type]
src\neo4j\_async\work\session.py:546: error: ParamSpec components are not allowed here  [valid-type]
src\neo4j\_async\work\session.py:547: error: ParamSpec components are not allowed here  [valid-type]

--- a/src/neo4j/_async/work/session.py
+++ b/src/neo4j/_async/work/session.py
@@ -543,8 +543,11 @@ class AsyncSession(AsyncWorkspace):
         transaction_function: t.Callable[
             te.Concatenate[AsyncManagedTransaction, _P], t.Awaitable[_R]
         ],
-        args: _P.args,
-        kwargs: _P.kwargs,
+        # *args: _P.args, **kwargs: _P.kwargs
+        # gives more type safety, but is less performant and makes for harder
+        # to read call sites
+        args: t.Any,
+        kwargs: t.Any,
     ) -> _R:
         self._check_state()
         if not callable(transaction_function):

--- a/src/neo4j/_sync/work/session.py
+++ b/src/neo4j/_sync/work/session.py
@@ -543,8 +543,11 @@ class Session(Workspace):
         transaction_function: t.Callable[
             te.Concatenate[ManagedTransaction, _P], t.Union[_R]
         ],
-        args: _P.args,
-        kwargs: _P.kwargs,
+        # *args: _P.args, **kwargs: _P.kwargs
+        # gives more type safety, but is less performant and makes for harder
+        # to read call sites
+        args: t.Any,
+        kwargs: t.Any,
     ) -> _R:
         self._check_state()
         if not callable(transaction_function):

